curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/convert-json-data-to-html.english.md

```diff
@@ -23,6 +23,7 @@ json.forEach(function(val) {
   html += "</div><br>";
 });
 ```
+
 <b>WARNING!</b> To complete this challenge you need to add new HTML elements. To do this you will use the innerHTML method, this can make your site vulnerable to XSS attacks, which in brief allows an attacker to execute code on the client side. To learn more <a href="https://guide.freecodecamp.org/security/xss-cross-site-scripting/">see the guide</a>
```

Comment:
I suggest using the verbiage of last suggestion made in the issue by @ojeytonwilliams .  It is much clearer.  Also, since the Guide will no longer be in the next release, the link to the Guide should be removed.  We prefer to use `Note:` instead of `Warning!` to be consistent with other challenges.
```suggestion
<b>Note:</b> For this challenge you need to add new html elements to the page, so you cannot rely on textContent. Instead you need to use innerHTML which can make a site vulnerable to cross site scripting attacks.
```

Reply:
Will update the spanish version to match